src/utils/constants.js

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable no-unused-vars */
```

Comment:
We add this file to drop major constants like the one below, which may be reusable.